scripts/e2e/deploy-merlin.sh

```diff
@@ -48,6 +48,8 @@ install_merlin() {
     --set imageBuilder.predictionJobBaseImages."3\.10\.*".mainAppPath=/home/spark/merlin-spark-app/main.py \
     --set ingress.host=merlin.caraml.${INGRESS_HOST} \
     --set mlflow.ingress.host=merlin-mlflow.caraml.${INGRESS_HOST} \
+    --set mlflow.image.repository=caraml-dev/mlflow \
+    --set mlflow.image.tag=1.26.1 \
```

Comment:
These tags need to be manually set for now since the Merlin helm chart hasn't been updated to use newer versions of Mlflow.